jenkins/migrationIntegPipelines/fullDefaultE2ETestCover.groovy

```diff
@@ -0,0 +1,9 @@
+def gitBranch = params.GIT_BRANCH ?: 'main'
```

Comment:
Why make a new Pipeline for this instead of use our existing ES 6.8 -> OS 2.X Pipeline?

Reply:
A few thoughts:
* Why wouldn't we want to use a pre-made snapshot for all of our integ tests?
* Why don't we want to test both RFS and C&R for each of our integ tests?
* `see if both pipelines or a single pipeline is failing` -> Can you elaborate a bit more on this one?

Reply:
I think its helpful for testing our workflow where the user performs the snapshot creation themselves from the migration console

In a similar vein, I like testing RFS and C&R together and separately as it affirms that these are separate independent pieces that can operate on their own, and that we haven't introduced some logic that say requires a snapshot to exist when we are only doing a C&R scenario

Reply:
Per discussion, my takeaways:
* Combine w/ the existing ES 6.8 test
* Update GitHub actions to use this instead of the existing 7.10 RFS only test

Reply:
Have updated this PR to include both these items. See recent Jenkins build here: https://migrations.ci.opensearch.org/job/full-es68source-e2e-test/4/